src/components/draggable/draggable-item.style.js

```diff
@@ -1,14 +1,28 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 import { padding, margin } from "styled-system";
-
 import { baseTheme } from "../../style/themes";
 import Icon from "../icon";
 
 const StyledDraggableContainer = styled.div`
+  ${({ as }) =>
+    as === "ul" &&
+    css`
+      padding-inline-start: 0;
+      padding-right: 4px;
+    `};
   ${margin}
 `;
 
 const StyledDraggableItem = styled.div`
+  ${({ as }) =>
+    as === "li" &&
+    css`
+      border-bottom: none !important;
```

Comment:
We want to avoid using `!important`, you should use the mechanism built in to styled-components to achieve this if possible

``` 
&& {
      border-bottom: none;
      &:not(:first-child) {
        margin-top: 8px;
      }
    }
```